### PR TITLE
feat: account and user avatar selector #1937 #1927

### PR DESCRIPTION
**Pull request by wlechowicz**
Adds Account view with avatar selector.

### CODE REVIEW

File: app/components/Account/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          <UserAvatarSelector
+            selectedAvatar={selectedAvatar}
+            setSelectedAvatar={setSelectedAvatar}
+          />
+          <FlexRow>
+            <Button primary onClick={() => alert('I wish I could')}>
+              Save
+            </Button>
+            <Button onClick={toggleIsEditingAvatar}>Cancel</Button>
+          </FlexRow>
```
Comment (rast-mk1) [inline app/components/Account/index.js:79]: When will `alert` be replaced with something else?

Comment (wlechowicz) [reply app/components/Account/index.js:79]: When there is backend for saving user avatar.

**Review by rast-mk1 — APPROVED** [review]

File: app/components/Account/index.js (hunk 1 above)
Comment (rast-mk1) [reply app/components/Account/index.js:79]: Okay, kinda lost track of what is ready concerning avatars.

Comment (wlechowicz) [reply app/components/Account/index.js:79]: Hint: check stories listed in PR title ;)

Comment (rast-mk1) [reply app/components/Account/index.js:79]: Didn't notice subtasks there, thanks!

**Review by weronika299 — APPROVED** [review]

**Review by marcelina-hasiak — APPROVED** [review]

**Review by hiimsagan — APPROVED** [review]

**Pull request merged**